### ISSUE TITLE
Updating to latest CMS version (3.2.31) causes Redirect dashboard to show 500 error

### ISSUE THREAD

**Issue by josephrlee** [riasvdv/statamic-redirect#41]
After I updated my local to the latest cms release, my redirect add on shows a 500 error on the Dashboard page and all my redirects disappear on the Redirect page.  

<img width="782" alt="Screen Shot 2022-01-14 at 2 46 06 PM" src="https://user-images.githubusercontent.com/1631062/149596136-d315e308-2445-4dd1-9f4a-c1ad0b9a9dbe.png">

<img width="1019" alt="Screen Shot 2022-01-14 at 2 47 33 PM" src="https://user-images.githubusercontent.com/1631062/149595132-90d8222b-a49a-49e1-bafa-8fd78e1a7eb8.png">

**Comment by riasvdv** [comment]
Could you share the full stacktrace of the exception?

**Comment by sunscreem** [comment]
@riasvdv Same here.


```
Symfony\Component\ErrorHandler\Error\FatalError Class Rias\StatamicRedirect\Stache\Errors\ErrorQueryBuilder contains 1 abstract method and must therefore be declared abstract or implement the remaining methods (Statamic\Stache\Query\Builder::getWhereColumnKeyValuesByIndex) 
    vendor/rias/statamic-redirect/src/Stache/Errors/ErrorQueryBuilder.php:49 [main]
```

Looks like its to do with caching as it only happened on production for me.

**Comment by sunscreem** [comment]
FYI - I rolled back to an older version of this and the same error occurs so I agree with the OP that it's probably in the update to Statamic that's causing this.

**Comment by riasvdv** [comment]
Should be fixed in 1.10.3 seems like the gents added a breaking change for addons that extend the query builder 😬
